### ISSUE TITLE
0.1.25  : Always get "Resolving URLCould not get a valid response from the SoundCloud API. Please check the API key"

### ISSUE THREAD

**Issue by ffroze** [Suyash458/soundcloud-dl#24]
I've got zero experience with coding, just gonna say that first, but yeah I'm using the client_id key under the network tab in the console on Chrome, I've tried other id keys and also checked the config file which showed the correct key. I'm on version 0.1.25, I've used the newest version of pip to verify I had all of the requirements installed. I have a working Go+ account and have tried different song links but get the same thing each time.

I'll admit it took me a few hours to figure out how to get it installed and such but I feel like I have a pretty decent understanding now and this error is really driving me crazy.

**Comment by suyashb95** [comment]
Whoops, looks like the `/resolve` endpoint works only on the newer soundcloud API. Could you try with v0.1.26?

**Comment by ffroze** [comment]
> Whoops, looks like the `/resolve` endpoint works only on the newer soundcloud API. Could you try with v0.1.26?

yeah I can try that, one question though sorry if this is dumb but how do I update to the newer version? 

edit: I got it updated and had one issue with enum34 but I uninstalled it and the api key set successfully, and finally it downloaded the track. One issue though is that it's only ripping the 128kbps mp3 and not the higher quality 256kbps m4a, is there something else I have to do to get it to do that? Also is it supposed to not have the metadata or album art?
